Packs/Expanse/Integrations/Expanse/Expanse.py

```diff
@@ -665,10 +666,13 @@ def fetch_incidents_command():
             behavior_incidents, next_offset = fetch_behavior_incidents_command(start_date, token, next_offset)
             for incident in behavior_incidents:
                 demisto.debug("Adding behavior incident name={name}, type={type}, severity={severity}".format(**incident))
-            demisto.incidents(behavior_incidents)
+            incidents += behavior_incidents
             if next_offset is None:
                 more_behavior = False
 
+    # Send all Incidents
+    demisto.incidents(incidents)
+
     # Save last_run
     demisto.setLastRun({
         "complete_for_today": True,
```

Comment:
So basically after one fetch you should be done for the day?

Reply:
Correct, we have two different APIs that can be used to provide incident details, and both are updated daily.